src/patternfly/patternfly-date-picker.scss

```diff
@@ -0,0 +1,177 @@
+// stylelint-disable
+.pf-c-date-picker__calendar {
+  &.flatpickr-calendar {
+    // calendar
+    --pf-c-date-picker__flatpickr__calendar--Background: var(--pf-global--BackgroundColor--light-100);
+    --pf-c-date-picker__flatpickr__calendar--Padding: var(--pf-global--spacer--md);
+    --pf-c-date-picker__flatpickr__calendar--open--ZIndex: var(--pf-global--ZIndex--sm);
```

Comment:
do you also need to repeat the z-index ?

Reply:
Yes, the calendar has it's own `z-index` declaration with higher specificity than `.pf-c-date-picker__calendar`